ci/yaml_to_json.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import yaml
+import json
+import os
+import fnmatch
+from collections import OrderedDict 
+
+# directory to store assets for test or release
+assets_dir = os.getcwd() + "/ci/assets"
+
+for file in os.listdir(assets_dir):
+  if fnmatch.fnmatch(file, '*index.yaml'):
+    with open(assets_dir + "/" + file, 'r') as s, open(assets_dir + "/" + os.path.splitext(file)[0] + ".json", "w") as o:
+        try:
+            doc = yaml.safe_load(s)
+            i = 0
+
+            if (doc['stacks'] != None):
+                for item in doc['stacks']:
+                    res = (OrderedDict([
+                        ("displayName", item['name']),
+                        ("description", item['description']),
+                        ("language", item['id']),
```

Comment:
language should be set to blank for now: "" as there is no concept of language in our stack.yamls

Reply:
Unfortunately Codewind does require a language.  Could we hardcode this for now to one of `java`, `nodejs`, `swift` accordingly?

Reply:
@makandre it is not possible for us to set these on the fly, unless we maintain a map of stacks and languages somewhere (quarkus -> java).

As per @tobespc, "" would suffice. Please, can you confirm?

Reply:
Toby's away but maybe @deboer-tim can comment?

Reply:
Although we can create a project w/o the language set, we will fail to display it in several places within the IDE as you can see in the screenshot:

![image](https://user-images.githubusercontent.com/17596240/62965581-63b3fd00-bdd3-11e9-9655-a1ffaad817cf.png)

It also breaks debugging.  Codewind uses the `language` to determine the style of debugging to use.

Reply:
Sorry, I don't know how the language is used and whether it gets through the template API to the client. I'm assuming at a minimum it would mean you can't filter the list of projects by language (which isn't necessarily a huge loss) but I don't if it would cause other problems. Might be worth trying (at least to see if it is ok for now), or pinging Howard.